app/views/unattended/snippets/_remote_execution_ssh_keys.erb

```diff
@@ -0,0 +1,16 @@
+<%#
+  kind: snippet
+  name: remote_execution_ssh_keys
+%>
+
+<% unless @host.params['remote_execution_ssh_keys'].blank? %>
+<% ssh_user = @host.params['remote_execution_ssh_user'] || 'root' %>
+
+mkdir -m=0700 -p ~<%= ssh_user %>/.ssh
```

Comment:
Exactly, maybe it should be even more generic, like `<%= plugin_snippets(:whatever) %>` and which would load all snippets with hook attribute set to `whatever`. This way we can add new extending points to templates just by changing the template. Only downside is 1.11+ dependency which might not be that bad.